src/objectTranslations/Contact-es.objectTranslation

```diff
@@ -3522,6 +3522,12 @@
         <name>Primary_Household__c</name>
         <relationshipLabel>Contactos (domicilio principal)</relationshipLabel>
     </fields>
+    <fields>
+        <help><!-- The language the contact prefers to use. --></help>
```

Comment:
Contact vs contact -- assume this comment for any instance like that in the help lines. Unless @juliesnow says differently, will defer to docs :)